rustworkx-core/src/lib.rs

```diff
@@ -73,6 +73,7 @@ pub type Result<T, E = Infallible> = core::result::Result<T, E>;
 pub mod bipartite_coloring;
 /// Module for centrality algorithms.
 pub mod centrality;
+pub mod collect_bicolor_runs;
```

Comment:
This was moved from under `pub mod connectivitiy` when I ran `cargo fmt`, I guess that they must be in alphabetical order (I will move it back to coloring algorithms).